cfpb/owning-a-home
# Fix JS errors: form explainer & GTM

There are a few jquery-related JS errors showing up on OaH pages: a `jquery is not defined` error related to GTM, and an occasional load issue with the jquery easing plugin that causes the next/prev page buttons to stop working on the form explainer pages in IE. A temporary fix for both issues is to load jquery in the head as the rest of the site does.

## Changes

- Load jquery in head 
- Update dependency versions
- Update form explainer tests

## Testing

- Check out branch
- Navigate to [http://localhost:8000/owning-a-home/closing-disclosure](http://localhost:8000/owning-a-home/closing-disclosure) in IE11
- Check that there is no `jQuery.easing[jQuery.easing.def] is not a function` or `Object doesn't support property or method 'undefined'` error in console when forward page button is clicked (try reloading the page several times; the error occurs randomly)

## Review
- @cfarm 
- @contolini 


## Screenshots

## Notes

- Once the GTM issues are resolved site-wide, we can go back to loading jquery with the rest of the js.
- The form explainer jquery easing issue should be fixed long-term by a) migrating to the cfgov-refresh version of expandables and/or b) using a react version of the form explainers.

## Checklist
